rp_transferto/utils.py

```diff
@@ -4,6 +4,19 @@
 import hmac
 import base64
 
+from prometheus_client import Summary
+
+transferto_request_time = Summary(
+    "transferto_request_time",
+    "request time for calls to transferto",
+    ["action"],
+)
+transferto_goods_and_services_request_time = Summary(
+    "transferto_goods_and_services_request_time",
+    "request time for calls to transferto goods and services API",
+    ["action"],
+)
```

Comment:
`Histogram`s are also useful for tracking request times, since that allows you to calculate percentiles

Reply:
Hmmm. Digging deeper in to [the docs](https://prometheus.io/docs/practices/histograms/), I think you're right.

> Two rules of thumb:

    If you need to aggregate, choose histograms.

    Otherwise, choose a histogram if you have an idea of the range and distribution of values that will be observed. Choose a summary if you need an accurate quantile, no matter what the range and distribution of the values is.